components/StyledCard.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export const StyledCard = styled.li`
+  border-radius: 8px;
+  gap: 10px;
+  margin: 10px;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: solid black 2px;
+`;
```

Comment:
you can move this styled component into your MyDonationList component and rename the latter one DonationCard (if you don't use this Card somewhere else)

Reply:
I use it in DonationListItem and in MyDonationList. So it better stays where it is...

Reply:
moved it :)